clients/admin-ui/cypress/support/stubs.ts

```diff
@@ -87,9 +87,19 @@ export const stubPlus = (available: boolean, options?: HealthCheck) => {
   }
 };
 
-export const stubHomePage = () => {
-  cy.intercept("GET", "/api/v1/privacy-request*", {
-    statusCode: 200,
-    body: { items: [], total: 0, page: 1, size: 25 },
-  }).as("getPrivacyRequests");
+export const stubHomePage = (hasConnections: boolean, hasSystems: boolean) => {
```

Comment:
nit: can we make this take a dict of props instead? otherwise reading `stubHomePage(true, true)` is hard to intuit what it means, vs `stubHomePage({hasConnections: true, hasSystems: true})`